### PR TITLE
Add interface to define the structure of the requested Gist

### PR DESCRIPTION
**Pull request by algomaster99**
Signed-off-by: Aman Sharma <mannu.poski10@gmail.com>

### CODE REVIEW

**Comment by algomaster99** [comment]
@domoritz apparently there is some issue with `express` typing as it is not able to accept `cors` and `session`. https://stackoverflow.com/a/62981871/11751642 Should we downgrade?

**Comment by domoritz** [comment]
Downgrade the typings? Sure. I'd not downgrade express itself if we can avoid it.

**Comment by algomaster99** [comment]
Yeah, we need to downgrade the typings. But this PR anyway can be included. It's adding an interface. I think you can merge it.

File: src/consts.ts [hunk 1]
```diff
@@ -4,3 +4,41 @@
  * _Exported as `paginationSize`_.
  */
 export const paginationSize = 8;
+
+/**
+ * Interface for defining structure of a received POST request
+ */
+export interface ICreateGist {
```
Comment (domoritz) [inline src/consts.ts:11]: Has someone else made these before where we could import them from?

Comment (algomaster99) [reply src/consts.ts:11]: I don't know. It's just the structure of the JSON data that we fetch from GitHub Gist so I thought to write my own custom one. 

Anyway, it's not in one of the libraries we already have. Also, downloading another package for this doesn't make sense.

**Comment by domoritz** [comment]
Go ahead and merge after reading me comment.

**Pull request merged**